dali/pipeline/operators/decoder/nvjpeg_decoder.h

```diff
@@ -127,6 +129,22 @@ class nvJPEGDecoder : public Operator<MixedBackend> {
     thread_pool_(max_streams_,
                  spec.GetArgument<int>("device_id"),
                  true /* pin threads */) {
+      const std::string cache_type = spec.GetArgument<std::string>("cache_type");
+      const std::size_t cache_size = static_cast<std::size_t>(
+        spec.GetArgument<int>("cache_size"));
+      const std::size_t cache_threshold = static_cast<std::size_t>(
+        spec.GetArgument<int>("cache_threshold"));
+      const bool cache_debug = spec.GetArgument<bool>("cache_debug");
+      if (cache_size > 0 && cache_size >= cache_threshold) {
```

Comment:
Maybe some enforce for `cache_size > 0`?

Reply:
Actually, cache_size = 0 means 'no cache' right now